pipeline-scripts/commonlib.groovy

```diff
@@ -16,12 +16,6 @@ ocpVersions = [
     "3.1",
 ]
 
-ocp4DefaultVersion = "4.0"
```

Comment:
You gotta rebase! You just removed what I just added.

Reply:
Yeah! sorry 'bout that.